MultiTable/src/RealtimeChangeHandler.js

```diff
@@ -22,8 +24,12 @@ function onSampleRemove(data, sample) {
     if (subjectGroup) {
       subjectGroup.removeSample(sample);
       const subject = subjectGroup.getSubjectForSample(sample);
-      const index = subject.samples.findIndex(s => (s.name === sample.name));
-      subject.samples.splice(index, 1);
+      if (subject) {
+        const index = subject.samples.findIndex(s => (s.name === sample.name));
```

Comment:
This is already being done in onSubjectAdd and onSubjectRemove. (this was the fix for the error the week after the release of the overflow splitting) The samples array is used elsewhere too so I think it's better to do the check when a subject is added or changed rather than every time subject.samples is accessed.

The other way a subject could be set with no samples array is if it is sent that way in the original hierarchy json. I think the hierarchy should always include an empty array, but I could also put this check in jsonToSubjectGroups to be safe.